<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tool</class>
 <widget class="QWidget" name="tool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>小工具库</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>121</width>
     <height>391</height>
    </rect>
   </property>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>20</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnBase64">
       <property name="text">
        <string>Base64</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMD5">
       <property name="text">
        <string>MD5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnURLEncoding">
       <property name="text">
        <string>URLEncoding</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widgetBase64" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>461</width>
     <height>391</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="txtBase64Input">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>461</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>请在此处输入待编码/解码内容</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtBase64Show">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>461</width>
       <height>150</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>编码/解码后的内容在此显示</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>329</y>
       <width>421</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnBase64Enc">
        <property name="text">
         <string>编码</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnBase64Dec">
        <property name="text">
         <string>解码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
